test/mpi/maint/render-topo.py

```diff
@@ -0,0 +1,64 @@
+import sys
+import re
+import argparse
```

Comment:
What are the purposes for these python scripts?

Reply:
These scripts are used to render the file dumped by MPIR_CVAR_COORDINATES_DUMP_FILE or MPIR_CVAR_TREE_DUMP_FILE_PREFIX, so the users can easily see the network topology and how the tree looks.

Reply:
Added the copyright and instructions.